src/rp2_common/hardware_sync/include/hardware/sync.h

```diff
@@ -95,11 +95,14 @@ typedef volatile uint32_t spin_lock_t;
 #define PICO_SPINLOCK_ID_CLAIM_FREE_FIRST 24
 #endif
 
-// PICO_CONFIG: PICO_SPINLOCK_ID_CLAIM_FREE_END, Highest Spinlock ID in the 'claim free' range, min=0, max=31, default=31, group=hardware_sync
-#ifndef PICO_SPINLOCK_ID_CLAIM_FREE_END
-#define PICO_SPINLOCK_ID_CLAIM_FREE_END 31
+#ifdef PICO_SPINLOCK_ID_CLAIM_FREE_END
+#warning PICO_SPINLOCK_ID_CLAIM_FREE_END has been renamed to PICO_SPINLOCK_ID_CLAIM_FREE_LAST
 #endif
 
+// PICO_CONFIG: PICO_SPINLOCK_ID_CLAIM_FREE_END, Highest Spinlock ID in the 'claim free' range, min=0, max=31, default=31, group=hardware_sync
```

Comment:
`PICO_SPINLOCK_ID_CLAIM_FREE_END` here needs to be `PICO_SPINLOCK_ID_CLAIM_FREE_LAST` :wink:

Reply:
damnit